utils/scripts/helpers/install_maldet.sh

```diff
@@ -0,0 +1,59 @@
+#!/bin/bash
```

Comment:
Is this part of the previous iteration that was using Docker and also no longer necessary?

Reply:
No.    The direction is obviously to be using Docker.